packages/builder/package.json

```diff
@@ -94,6 +94,7 @@
     "@sveltejs/vite-plugin-svelte": "1.4.0",
     "@testing-library/jest-dom": "6.4.2",
     "@testing-library/svelte": "^4.1.0",
+    "@types/shortid": "^2.2.0",
```

Comment:
Had to add types for shortid as it was complaining, so the lockfile changed massively. In case anyone was wondering about the PR size